Sources/ApolloCodegenLib/Templates/SwiftPackageManagerModuleTemplate.swift

```diff
@@ -16,7 +16,7 @@ struct SwiftPackageManagerModuleTemplate: TemplateRenderer {
     let casedSchemaName = config.schemaName.firstUppercased
 
     return TemplateString("""
-    // swift-tools-version:5.6
+    // swift-tools-version:5.7
```

Comment:
There's nothing in `ApolloAPI` at the moment that would warrant this but there is no reason we couldn't use those new Swift features there.